src/widgets/content/model.rs

```diff
@@ -11,11 +11,13 @@ pub struct ContentModel {
 	pub task_details_factory: AsyncFactoryVecDeque<TaskDetailsFactoryModel>,
 	pub task_entry: Controller<TaskInputModel>,
 	pub parent_list: Option<SidebarList>,
-	pub compact: bool,
 	pub icon: Option<String>,
 	pub title: String,
 	pub description: String,
 	pub smart: bool,
 	pub selected_task: Option<Task>,
 	pub show_task_details: bool,
+	pub page_icon: String,
```

Comment:
Is there a reason why this can't be `&'static str`?

Reply:
No reason at all 👍🏼